### PR TITLE
Docs 3237 vpc licensing explanation

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

File: modules/ROOT/pages/virtual-private-cloud.adoc [hunk 1]
```diff
@@ -17,6 +17,30 @@ Each VPC allows you to configure firewall rules to apply to your workers. You ca
 Additionally, you can configure one or more dedicated load balancers to validate requests using your own SSL certificates, and to map input URLs to call to different CloudHub applications. +
 Combining firewall rules and dedicated load balancers configurations, you can fine tune how external services reach your applications.
 
+[[example_deployment_scenarios]]
+== Example VPC Deployment Scenarios
+When you are planning and configuring your your CloudHub VPCs, you should make sure you have enough licenses to meet your requirements.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/virtual-private-cloud.adoc]: nit: `your your`

File: modules/ROOT/pages/virtual-private-cloud.adoc [hunk 2]
```diff
@@ -17,6 +17,30 @@ Each VPC allows you to configure firewall rules to apply to your workers. You ca
 Additionally, you can configure one or more dedicated load balancers to validate requests using your own SSL certificates, and to map input URLs to call to different CloudHub applications. +
 Combining firewall rules and dedicated load balancers configurations, you can fine tune how external services reach your applications.
 
+[[example_deployment_scenarios]]
+== Example VPC Deployment Scenarios
+When you are planning and configuring your your CloudHub VPCs, you should make sure you have enough licenses to meet your requirements. 
+
+=== 1:1 Relationship Deployment
+
+In this scenario, you are using a 1:1 relationship, which means that each VPC can connect to a single public IP address at a remote location.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/virtual-private-cloud.adoc]: So how many licenses am I consuming? (Should be 1, right?)

File: modules/ROOT/pages/virtual-private-cloud.adoc [hunk 3]
```diff
@@ -17,6 +17,30 @@ Each VPC allows you to configure firewall rules to apply to your workers. You ca
...
+
+image::vpc-1-1-deployment.png[1:1 VPC Deployment]
+
+=== Connect a Single VPC to Two Remote Locations
+
+A strict 1:1 relationship between a VPC and remote location is not necessary. In this example, a single VPC is connected to public IP addresses at two remote locations. This means that you are using both of the VPC licenses that are included in the base VPC subscription for a single VPC, which leaves no licenses for connectivity to the second VPC. 
+
+image::vpc-one-to-two-remote-locations.png[One VPC Connecting to Two Remote Locations]
+
+=== Connect Two VPCs to Two Remote Locations
```
Comment (sebastiankorol) [inline modules/ROOT/pages/virtual-private-cloud.adoc]: Should we say "to Two Remote Locations Each"?

File: modules/ROOT/pages/virtual-private-cloud.adoc (hunk 1 above)
Comment (valkyrie69) [reply modules/ROOT/pages/virtual-private-cloud.adoc]: I love nits. :-)

File: modules/ROOT/pages/virtual-private-cloud.adoc [hunk 4]
```diff
@@ -17,6 +17,30 @@ Each VPC allows you to configure firewall rules to apply to your workers. You ca
...
+
+If you implement a VPC deployment scenarious where both VPCs connect to two remote locations, four VPC licenses are required, even if you are using the same two public IP addresses. This is because each remote location that needs to be accessible from the Cloudhub VPC requires an additional VPC license.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/virtual-private-cloud.adoc]: nit: scenario

File: modules/ROOT/pages/vpc-provisioning-concept.adoc [hunk 5]
```diff
@@ -6,7 +6,9 @@ endif::[]
 Provisioning a VPC for your organization consists of two major steps:
 
 . Create the VPC. +
-This is a self-service process. You can create a VPC using either the Runtime Manager UI, or the Anypoint Platform CLI.
+This is a self-service process. You can create a VPC using either the Runtime Manager UI, or the Anypoint Platform CLI. +
```
Comment (sebastiankorol) [inline modules/ROOT/pages/vpc-provisioning-concept.adoc:9]: I don't know the current state, but is this true today? I thought Support needs to enable self-service first

Comment (sebastiankorol) [reply modules/ROOT/pages/vpc-provisioning-concept.adoc:9]: NVM I just realized this is about VPCs, not VPNs

Comment (valkyrie69) [reply modules/ROOT/pages/vpc-provisioning-concept.adoc:9]: You can create a VPC (I've done it with the UI but don't know about CLI--I assume you can there too) but support has to create the VPNs.

File: modules/ROOT/pages/vpc-provisioning-concept.adoc [hunk 6]
```diff
@@ -28,6 +30,15 @@ To enable the VPC to host your workers, it must be bound to a business group and
 
 Read the following FAQs to understand how to plan a successful VPC setup.
 
+== FAQ: How to Calculate VPC and VPN Licensing Requirements
+
+To connect a CloudHub VPC through IPSec VPN, you must have the required number of licenses for your deployment scenario. 
+
+The Anypoint Platform base subscription includes two VPC licenses, which entitles your Anypoint Platform organization to two VPCs. VPN licenses are bundled with the VPC license, so each VPC license also entitles the organisation to one VPN gateway with a connection to one public IP address in a remote location. Therefore, one VPC License = 1 VPC entitlement + 1 VPN entitlement.
```
Comment (sebastiankorol) [inline modules/ROOT/pages/vpc-provisioning-concept.adoc]: Do we want to call out that the VPN entitlement doesn't need to be used 1:1 with the VPC entitlement? E.g. if I have two VPCs, I can have two VPNs in a VPC, and none in the other VPC

File: modules/ROOT/pages/virtual-private-cloud.adoc [hunk 7]
```diff
@@ -7,7 +7,7 @@ The Virtual Private Cloud (VPC) offering allows you to create a virtual, private
 
 Connecting to your CloudHub VPC extends your corporate intranet and allows CloudHub workers to access resources behind your corporate firewall. You can connect on-premises data centers through a secured VPN tunnel, or a private AWS VPC through VPC peering, or by using AWS Direct Connect.
 
-The base VPC subscription includes two VPCs and each VPC can be associated with multiple environments. This allows you, for example, to have one isolated network for your production environment, and another for your QA and staging.
+The base VPC subscription includes two VPCs and each VPC can be associated with multiple environments. This allows you, for example, to have one isolated network for your production environment, and another for your QA and staging. Your licensing requirements will depend on your specific deployment scenario.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/virtual-private-cloud.adoc]: remove "will"

File: modules/ROOT/pages/virtual-private-cloud.adoc [hunk 8]
```diff
@@ -17,6 +17,30 @@ Each VPC allows you to configure firewall rules to apply to your workers. You ca
 Additionally, you can configure one or more dedicated load balancers to validate requests using your own SSL certificates, and to map input URLs to call to different CloudHub applications. +
 Combining firewall rules and dedicated load balancers configurations, you can fine tune how external services reach your applications.
 
+[[example_deployment_scenarios]]
+== Example VPC Deployment Scenarios
+When you are planning and configuring your CloudHub VPCs, you should make sure you have enough licenses to meet your requirements.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/virtual-private-cloud.adoc]: I think it's Anypoint VPC? Or do we mean VPC for CloudHub specifically?

File: modules/ROOT/pages/vpc-provisioning-concept.adoc [hunk 9]
```diff
@@ -28,6 +30,18 @@ To enable the VPC to host your workers, it must be bound to a business group and
...
 Read the following FAQs to understand how to plan a successful VPC setup.
 
+== FAQ: How to Calculate VPC and VPN Licensing Requirements
+
+To connect a CloudHub VPC through IPSec VPN, you must have the required number of licenses for your deployment scenario. 
+
+The Anypoint Platform base subscription includes two VPC licenses, which entitles your Anypoint Platform organization to two licensed VPCs. VPN licenses are bundled with the VPC license, so each VPC license also entitles the organization to one VPN gateway with a connection to one public IP address in a remote location. 
+
+[NOTE]
+VPN licenses do not need to have a one-to-one relationship with the VPC licenses. For example, you can have two VPCs with two VPNs in one of them and none in the other.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/vpc-provisioning-concept.adoc]: "For example, you can have one VPC with two VPNs and no VPNs in the other VPC."?  "in one of them" sounds odd to me.

File: modules/ROOT/pages/vpc-provisioning-concept.adoc [hunk 10]
```diff
@@ -28,6 +30,18 @@ To enable the VPC to host your workers, it must be bound to a business group and
...
+== FAQ: How to Calculate VPC and VPN Licensing Requirements
+
+To connect a CloudHub VPC through IPSec VPN, you must have the required number of licenses for your deployment scenario. 
+
+The Anypoint Platform base subscription includes two VPC licenses, which entitles your Anypoint Platform organization to two licensed VPCs. VPN licenses are bundled with the VPC license, so each VPC license also entitles the organization to one VPN gateway with a connection to one public IP address in a remote location. 
+
+[NOTE]
+VPN licenses do not need to have a one-to-one relationship with the VPC licenses. For example, you can have two VPCs with two VPNs in one of them and none in the other. 
+
+An additional VPC license is required for each remote location that needs to be accessible from the CloudHub VPC.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/vpc-provisioning-concept.adoc]: How is a CloudHub VPC different from Anypoint VPC?

**Review by Mulesoft-Mysti — APPROVED** [review]
Some suggested changes.

**Pull request merged**